SConstruct

```diff
@@ -447,22 +472,23 @@ elif platform == "win32":
 					break
 	# Extra: Microsoft redistributable libraries installer
 	if 'msvc' in env["TOOLS"]:
-		if path.exists("dep/VCRedistInstall.exe"):
+		if path.exists("deps/VCRedistInstall.exe"):
 			env.Install("build/Blades of Exile/", "dep/VCRedistInstall.exe")
 		else:
 			print("WARNING: Cannot find installer for the MSVC redistributable libraries for your version of Visual Studio.")
 			print("Please download it from Microsoft's website and place it at:")
-			print("      dep/VCRedistInstall.exe")
+			print("      deps/VCRedistInstall.exe")
 			# Create it so its lack doesn't cause makensis to break
 			# (Because the installer is an optional component.)
+			os.makedirs("build/Blades of Exile", exist_ok=True)
 			open("build/Blades of Exile/VCRedistInstall.exe", 'w').close()
 
 if platform == "darwin":
 	env.VariantDir("#build/pkg", "pkg/mac")
 	SConscript("build/pkg/SConscript")
-elif platform == "win32" and subprocess.call(['where', '/Q', 'makensis']) == 0:
-	env.VariantDir("#build/pkg", "pkg/win")
-	SConscript("build/pkg/SConscript")
+#elif platform == "win32" and subprocess.call(['where', '/Q', 'makensis']) == 0:
```

Comment:
This needs to be uncommented or otherwise fixed before I can merge it.

Reply:
The installer part is having weird problems. I didn't want to deal with it because I think an auto-updating deployment target like Itch or Steam is preferable to having an installer at all.

Reply:
Putting it on Itch or Steam is fine, but we should support a standalone installer build as well to support people who don't like using services like Itch or Steam. What kind of weird problems was it having? Maybe there's an issue of using deprecated NSIS stuff or something…